my_code.py

```diff
@@ -1,15 +1,26 @@
 # Collaborators (including web sites where you got help: (enter none if you didn't need help)
 #  
 
-def factorial_calc(x):   #you may choose the name of the parameter
+def factorial_calc(num):   #you may choose the name of the parameter
 
-    return    # be sure to return the factorial
+    factorial = 1
 
+    if num < 0:
+        print("Factorial does not exist for any negative numbers.")
+    elif num == 0:
+        print("The factorial of 0 is 1.")
+    else:
+        for x in range(1, num + 1):
+            factorial = x * 1
```

Comment:
factorial = x * factorial (each time through, you need to multiply the previous product by the current loop number)